numba/tests/test_withlifting.py

```diff
@@ -522,15 +522,21 @@ def test_case07_mystery_key_error(self):
         def foo(x):
             with objmode_context():
                 t = {'a': x}
-            return x, t
+                u = 3
+            return x, t, u
         x = np.array([1, 2, 3])
         cfoo = njit(foo)
+
         with self.assertRaises(errors.TypingError) as raises:
             cfoo(x)
+
+        exstr = str(raises.exception)
         self.assertIn(
-            "missing type annotation on outgoing variables",
-            str(raises.exception),
-            )
+            "Missing type annotation on outgoing variable(s): ['t', 'u']",
+            exstr)
+        self.assertIn("Example code: with objmode"
+                      "(t='<add_type_as_string_here>')",
+            exstr)
```

Comment:
odd indentation here---i didn't see `exstr` at first.  can the string "Example code..." be move to the same level as `exstr`?

Reply:
done in 210a8f2